sdk/src/compressed_account.rs

```diff
@@ -25,15 +23,19 @@ pub trait LightAccounts: Sized {
         address_merkle_tree_root_index: u16,
         remaining_accounts: &[AccountInfo],
     ) -> Result<Self>;
-    fn new_address_params(&self) -> Vec<NewAddressParamsPacked>;
+    fn new_address_params(&self) -> Vec<light_system_program::NewAddressParamsPacked>;
```

Comment:
doesn't use the sdk type

Reply:
Not yet. In this PR, I'm just defining the types, not using them in the programs. If programs switch to the new types now, there will be an incompatibility between types returned by `TestIndexer` (which still uses the ones from `light_system_program`) and types used in program.

So I'm going to switch all the types in the last step, when the new `PhotonIndexer` is ready and I can ditch the `TextIndexer`.